config.go

```diff
@@ -45,6 +45,8 @@ type Config struct {
 	LogOutput io.Writer
 	// HTTPClient is the http client
 	HTTPClient *http.Client
+	// Specify an Explicit Marathon Version
+	Version string
```

Comment:
Teaching go-marathon to understand the Marathon version it's talking to (either manually or automatically) in a reliable way introduces a certain layer of complexity with a number of corner cases that we should consider carefully. This probably deserves a separate issue and discussion, so I'd rather avoid extending the consumer API surface along this PR by something we don't have a clear usage pattern for yet.

I believe the amount of test code we'd need to update in order for the signature change to happen should be fairly manageable: We have 4 or 5 call sites (including the new one from this PR) where the existing `config` parameter is not nil; those would need to be modified to pass a new configuration container struct instead. The internal change to `newFakeMarathonEndpoint` should be very simple too. If you prefer to minimize the overall effort, we could also create a new function `newFakeMarathonEndpointFromConfigs(Config, ServerConfig)` that you can call directly from the new test and the existing `newFakeMarathonEndpoint` (with the `ServerConfig` argument being nil for the latter case). A follow-up PR could then clean up all remaining call sites.